libs/auth-api-lib/migrations/20220915153658-add-delegation-domain.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface) {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+        ALTER TABLE delegation
+          ADD COLUMN domain_name VARCHAR DEFAULT '@island.is' NOT NULL;
+    
+        ALTER TABLE delegation
+            ADD CONSTRAINT fk_delegation_domain FOREIGN KEY (domain_name) REFERENCES domain (name);
+        
+        CREATE INDEX delegation__domain_name__index
+            ON delegation (domain_name);
+        
+        ALTER TABLE delegation
+          DROP CONSTRAINT IF EXISTS unique_from_to;
```

Comment:
Should be mirrored in `down` to not break `up => down => up` flow?

Reply:
So delete rows with domain different from the default and then add constraint?

Reply:
Ah sorry, I actually missed the "IF EXISTS". I just don't want the flow to crash the release but sounds like this should work (and better to not remove data).